x-pack/metricbeat/module/aws/ec2/ec2.go

```diff
@@ -44,6 +45,13 @@ func init() {
 // interface methods except for Fetch.
 type MetricSet struct {
 	*aws.MetricSet
+	Tags []Tag `config:"tags_filter"`
```

Comment:
No need to change this, just curiosity. Have you considered using a `map[string]string` for these filter tags?

Reply:
Yeah good point, I went to Tag struct directly because that's how aws defined their tags but didn't think about map[string]string. I will think about it to see if it will make the code simpler.

Reply:
No need to change it, I don't think an option is really better than the other. Having it as an array of structs is more flexible, for example if each tag can have some kind of option in the future. If they won't, probably a map is simpler.